templates/workspace_services/databricks/parameters.json

```diff
@@ -0,0 +1,56 @@
+{
+  "schemaVersion": "1.0.0-DRAFT+TODO",
+  "name": "tre-service-databricks",
+  "created": "2021-06-03T11:54:54.0225968Z",
+  "modified": "2021-06-03T11:54:54.0225968Z",
```

Comment:
This isn't the updated format. Check the file under the base workspace for an example.

Reply:
aligned with the new format